tools/WixBuild.props

```diff
@@ -184,7 +192,22 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VCPath>$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC', 'ProductDir', null, RegistryView.Registry64, RegistryView.Registry32))</VCPath>
+    <VS2017Path Condition=" '$(VS2017Path)'=='' and '$(VsInstallRoot)'!=''">$(VsInstallRoot)\</VS2017Path>
```

Comment:
Maybe beyond the scope right now, but a task can be added to find the latest instance with the `Microsoft.VisualStudio.Workload.NativeDesktop`. I hope to get around to one eventually (other projects in the works working up to it), but - like with https://github.com/Microsoft/vssetup.powersell - would effectively look like:

```powershell
$path = get-vssetupinstance | select-vssetupinstance -version '[15.0,16.0)' -require Microsoft.VisualStudio.Workload.NativeDesktop -latest | select-object -expand InstallationPath
```